erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -447,6 +446,12 @@ def set_operation_status(self, update=False, update_modified=True):
 				'total_operating_cost': self.total_operating_cost,
 			}, update_modified=update_modified)
 
+	def validate_completed_qty_in_operations(self):
+		max_production_qty = flt(self.get_qty_with_allowance(self.producible_qty), self.precision("qty"))
+		for d in self.operations:
+			if flt(d.completed_qty) > max_production_qty:
+				frappe.throw(_("Completed Qty can not be greater than 'Qty to Produce'"))
```

Comment:
Lets add Operation name in the error message
Completed Qty for Operation {0} can not be greater than 'Qty to Produce'